app/ui/browser-modern/views/browser/location/index.jsx

```diff
@@ -68,11 +68,11 @@ class Location extends Component {
 
   handleInputKeyDown = e => {
     if (e.key === 'Enter') {
-      this.handleAutocompletionSelect({ uri: e.target.value });
+      this.handleAutocompletionPick({ uri: e.target.value });
     }
   }
 
-  handleAutocompletionSelect = data => {
+  handleAutocompletionPick = ({ data }) => {
```

Comment:
I strongly disagree. When you hover over autocompletions or history items, they're "selected" highlighted. Only when you click on them or press enter they become "chosen", or "picked". I don't care about what other word we use, but "select" is poor because it implies this function is called when list items are hovered.